### DISCUSSION TITLE
https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz not reachable

### DISCUSSION THREAD

**Discussion by danielchristianschroeter** [okd-project/okd#934] [Q&A] [answered]
Is the link https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz under https://docs.okd.io/latest/cli_reference/openshift_cli/getting-started-cli.html still valid?
There is no response from this site anymore.

**Comment by vrutkovs** [comment]
https://status.redhat.com/incidents/507g1kpnptr5

**Reply by clacsinaRX-M** [reply]
Despite this fix I still cannot seem to access the `oc.tar.gz` at https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz 

To me it appears the file itself is missing from the mirror site:

![image](https://user-images.githubusercontent.com/43353000/138140028-a645bceb-0cd3-498f-a2a8-fa523667e7d2.png)

**Reply by vrutkovs** [reply]
Oops, that's a different problem. https://mirror2.openshift.com/pub/openshift-v4/amd64/clients/ocp/latest/ should have it (probably this subdomain will go away later though)

**Comment by vrutkovs** [comment, marked as answer]
This location is deprecated - https://mirror.openshift.com/pub/openshift-v4/clients/oc/$__DEPRECATED_LOCATION__PLEASE_READ__.txt

https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/ should be the new one. At the moment its empty, but should be back soon
